app.js

```diff
@@ -12,14 +14,146 @@ function toggleMobileMenu() {
-
 // Close function for mobile-menu
 function closeMobileMenu() {
   mobileMenuList.classList.remove('show');
   mobileMenuButton.classList.remove('hidden');
 }
+mobileMenuButton.addEventListener('click', toggleMobileMenu);
+closeMenu.addEventListener('click', closeMobileMenu);
+
+const portfolioItemsDetail = [
+  {
+    id: '0',
+    name: 'Tonic',
+    year: '2015',
+    text: 'Lorem  of type and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'CANOPY',
+    stack: 'Back End Dev',
+    workStack: ['HTML', 'CSS', 'JavaScript'],
+    images: 'images/natureDesktop.png',
+    source: '#',
+    live: '#',
+  },
+
+  {
+    id: '1',
+    name: 'Multi-Post Stories',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of the printing and  Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'FACEBOOK',
+    stack: 'Full Stack Dev',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/storiesDesktop.png',
+    source: '#',
+    live: '#',
+  },
 
-// An event listener
-closeMobileMenu.addEventListener('click', closeMobileMenu);
+  {
+    id: '2',
+    name: 'Facebook 360',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'Facebook',
+    stack: 'Full Stack Dev',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/Facebook360Desktop.png',
+    source: '#',
+    live: '#',
+  },
+
+  {
+    id: '3',
+    name: 'Uber Navigation',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of a galley of type and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'Uber',
+    stack: 'Lead Developer',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/UberDesktop.png',
+    source: '#',
+    live: '#',
+  },
+];
+if (portfolio) {
+  for (let i = 0; i < portfolioItemsDetail.length; i += 1) {
+    const div = document.createElement('div');
+    div.classList.add('card');
+    div.setAttribute('id', `${i}`);
+    div.innerHTML = `
+   <div class="card-left-mobile">
+   <img src="${portfolioItemsDetail[i].images}" alt="snap-card">
+   </div>
+   <div class="card-right">
+     <h2>${portfolioItemsDetail[i].name}</h2>
+     <ul class="cano">
+       <li>${portfolioItemsDetail[i].title}<i class="fa fa-circle"></i></li>
+       <li>${portfolioItemsDetail[i].stack}<i class="fa fa-circle"></i></li>
+       <li>2015</li>
+     </ul>
+     <p>${portfolioItemsDetail[i].text}</p>
+     <ul class="lang">
+       ${portfolioItemsDetail[i].workStack
+    .map((stack) => `<li>${stack}</li>`)
+    .join('')}
+     </ul>
+     <button class="btn-default portfolio-items-detail">See project</button>
+   </div>
+   <div class="card-left-desk">
```

Comment:
- [ ] As suggested in the ️[project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md), kindly try as much as possible to stick to the Figma original design. For this case, I suggest that you consider displaying all the contents of the popup card as seen in the Figma design ☺️. 
- [ ] In addition, I suggest that you reduce the blur effect to ensure that the background contents are blurry seen 👍.

Figma design 😎 
![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/99394374/b14a990c-c442-40ff-af93-ddd9b8e3493a)

Your design 😎 
![image](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/99394374/87ae9b73-c890-4bc5-b563-fc90da78f963)